src/components/Form/Autocomplete.js

```diff
@@ -44,9 +44,8 @@ const Autocomplete = ({
           <div {...styles.root}>
             <Inner isOpen={isOpen}>
               <Field
-                isFocused={isOpen}
                 label={label}
-                value={filter}
+                value={isOpen ? filter : value.text}
```

Comment:
```
TypeError: Cannot read property 'text' of null
```

Happens on the initial load of the styleguide doc page.

Reply:
Phew. Will fix asap.